spark/src/test/resources/cog-layer/attributes/multiband-cog-layer__.__0__.__metadata.json

```diff
@@ -0,0 +1 @@
+[{"name":"multiband-cog-layer","zoom":0},{"header":{"format":"file","path":"spark/src/test/resources/cog-layer","layerType":"COGLayerType","keyClass":"geotrellis.spark.SpatialKey","valueClass":"geotrellis.raster.MultibandTile"},"metadata":{"metadata":{"zoomRangesInfos":[[{"minZoom":0,"maxZoom":0},{"minKey":{"col":0,"row":0},"maxKey":{"col":0,"row":0}}],[{"minZoom":1,"maxZoom":1},{"minKey":{"col":1,"row":0},"maxKey":{"col":1,"row":0}}],[{"minZoom":2,"maxZoom":2},{"minKey":{"col":2,"row":1},"maxKey":{"col":2,"row":1}}],[{"minZoom":3,"maxZoom":3},{"minKey":{"col":5,"row":3},"maxKey":{"col":5,"row":3}}],[{"minZoom":4,"maxZoom":4},{"minKey":{"col":11,"row":7},"maxKey":{"col":11,"row":7}}],[{"minZoom":5,"maxZoom":5},{"minKey":{"col":23,"row":14},"maxKey":{"col":23,"row":15}}],[{"minZoom":6,"maxZoom":6},{"minKey":{"col":46,"row":29},"maxKey":{"col":46,"row":30}}],[{"minZoom":7,"maxZoom":7},{"minKey":{"col":92,"row":59},"maxKey":{"col":92,"row":60}}],[{"minZoom":8,"maxZoom":11},{"minKey":{"col":184,"row":118},"maxKey":{"col":185,"row":120}}]],"extent":{"xmin":80.0,"ymin":5.0,"xmax":81.0,"ymax":7.0},"layoutScheme":{"crs":"+proj=longlat +datum=WGS84 +no_defs ","tileSize":256,"resolutionThreshold":0.1},"cellType":"int16","crs":"+proj=longlat +datum=WGS84 +no_defs "},"keyIndexes":[[{"minZoom":0,"maxZoom":0},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":0,"row":0},"maxKey":{"col":0,"row":0}}}}],[{"minZoom":7,"maxZoom":7},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":92,"row":59},"maxKey":{"col":92,"row":60}}}}],[{"minZoom":6,"maxZoom":6},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":46,"row":29},"maxKey":{"col":46,"row":30}}}}],[{"minZoom":4,"maxZoom":4},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":11,"row":7},"maxKey":{"col":11,"row":7}}}}],[{"minZoom":8,"maxZoom":11},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":184,"row":118},"maxKey":{"col":185,"row":120}}}}],[{"minZoom":1,"maxZoom":1},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":1,"row":0},"maxKey":{"col":1,"row":0}}}}],[{"minZoom":2,"maxZoom":2},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":2,"row":1},"maxKey":{"col":2,"row":1}}}}],[{"minZoom":5,"maxZoom":5},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":23,"row":14},"maxKey":{"col":23,"row":15}}}}],[{"minZoom":3,"maxZoom":3},{"type":"zorder","properties":{"keyBounds":{"minKey":{"col":5,"row":3},"maxKey":{"col":5,"row":3}}}}]]}}]
```

Comment:
So you decided not to zip this layer?

Reply:
Yeah, but only because that's what we seemed to be doing with all of the other test files in that directory